packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -302,7 +302,12 @@ function _SignUpStart(): JSX.Element {
       </Control>
     ) : null;
 
-  const atLeastOneFormField = nameField || usernameField || emailAddressField || phoneNumberField || passwordField;
+  const atLeastOneAuthFactor =
```

Comment:
❓ Would it make sense to extract this as a `hasValidAuthFactor` method on the `UserSettings` class?

Eg: 
```tsx
  get hasValidAuthFactor() {
    return this.attributes.email_address.used_for_first_factor ||
    this.attributes.phone_number.used_for_first_factor ||
    (this.attributes.password.required && this.attributes.username.required);
  }

```